README.md

```diff
@@ -1 +1,148 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Book Store] <a name="about-project"></a>
+
+**Book Store**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/BB-Simon/">yml for linters</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JS</a></li>
+    <li><a href="https://legacy.reactjs.org/">ReactJS</a></li>
+    <li><a href="https://github.com">GIT/Github</a></li>
+    <li><a href="https://nodejs.org/en/">NodeJS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **[Create react app by create-react-app]**
+- **[Add githubflow]**
+- **[Add linters configeration]**
+- **[Add browser routes]**
+- **[Add Navigation component]**
+- **[Add Home page]**
+- **[Add Categories page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- VSCode or any other IDE.
+- node.js(preferable).
+- browser to deploy.
+- GitHub/Git.
+
+### Setup
+
+> Clone this repository to your desired folder:
+
+git clone https://github.com/BB-Simon/book-store
+
+> In the project directory, you can run:
+
+to install dependencies:
+#### `npm install`
+
+to run the project:
+#### `npm start`
+
+to build:
+#### `npm build`
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤  **BB-Simon**
+
+- GitHub: [@githubhandle](https://github.com/BB-Simon) 
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bb-simon/) 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Refactor code with classes**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/BB-Simon/book-store/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you are welcomed to star and fork it. You can also share your ideas for future features. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for their idea of  the idea of Book Store project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] You did a great job working on your project documentation! 🚀 However, please make sure your LICENSE link is working! 🤓 🔧